### PR TITLE
configure circle ci to run test parallel, reducing test run time

### PR DESCRIPTION
**Pull request by Tieske**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ dependencies:
         - sudo make dev
 test:
     override:
-        - busted -o spec/busted-print.lua --coverage spec/
+        - busted -v -o spec/busted-print.lua --coverage :
+            parallel: true
+            files:
+              - spec/**/*_spec.lua
```
Comment (thibaultCha) [inline circle.yml]: Is this necessary for parallel? Since it's already the default for busted.

Comment (Tieske) [inline circle.yml]: yes. If you don't it runs once and displays the result in all four containers.

Check [this one](https://circleci.com/gh/Mashape/kong/914) the busted command at the top gets explicit files to test. CircleCI build 1 image, copies it to 4 and splits the test files over the 4 images (appending the filenames to the busted command)
.

Comment (Tieske) [inline circle.yml]: The others have a warning displayed; "Warning: Your build is configured to use 4 containers in parallel, but none of your test actions are configured to run in parallel. Consider either lowering your build parallelism, or parallelizing your tests steps."

**Comment by thibaultCha** [comment]
Tests are failing but for an unrelated reason fixed on `next`. I guess this is ok!

**Pull request merged**